ONSdigital/ssdc-rm-caseprocessor
# Use fully qualified Pub/Sub topic and subscription names

# Motivation and Context
Having completely separate config for the PubSubTemplate for our project, shared project and emulator too, created a lot of config code to have to manage. It turns out that it was nearly all redundant: we just needed to use fully qualified topic and subscription names, to specify specific projects.

# What has changed
Refactored to use fully qualified names of topics and subscriptions, where they exist outside our project.

# How to test?
Zero regression in any of the tests. Can connect to your GCP project too, if you fiddle with the config to point at your GCP project (see README).

# Links
Trello: https://trello.com/c/Wp6A6wc2
